src/ut/dns_json/multiple_rrtypes_for_name.json

```diff
@@ -0,0 +1,21 @@
+{
+  "hostnames": [
+    {
+      "name": "one.redirected.domain",
+      "records": [
+        {
+          "target": "one.made.up.domain"
```

Comment:
I'm guessing this is just in because it was copied from the bad file, and shouldn't cause a problem anyway so left in. That's fine I think